lockfile_test.go

```diff
@@ -1,28 +1,112 @@
-package lockfile_test
+package lockfile
 
 import (
-	lockfile "."
 	"fmt"
-	"os"
+	"io/ioutil"
+	"math/rand"
 	"path/filepath"
+	"strconv"
+	"testing"
 )
 
-func ExampleLockfile() {
```

Comment:
Could you keep the example? I consider an automatically verified example that can be copy'n'pasted as rather important for a package.